jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -743,13 +743,13 @@ private static void ensureProperties(boolean isInitialization) {
 		}
 		initializedProperties.put(key, list[i+1]);
 	}
-	/*[IF JAVA_SPEC_VERSION == 11]*/
+	/*[IF (JAVA_SPEC_VERSION <= 11) & Sidecar18-SE-OpenJ9]*/
```

Comment:
`(JAVA_SPEC_VERSION <= 11)` is redundant here (in the else part of `JAVA_SPEC_VERSION >= 17`).